### ISSUE TITLE
ShadowReader: Fix accessibility issues

### ISSUE THREAD

**Issue by pbakaus** [ampproject/amp-publisher-sample#71]
The Shadow Reader currently cannot be fully navigated via keyboard, and has probably plenty of other accessibility issues.

**Comment by pbakaus** [comment]
Will be fixed via #75.
